crates/external_services/src/kms.rs

```diff
@@ -96,11 +96,52 @@ impl KmsClient {
 
         Ok(output)
     }
+
+    /// Encrypts the provided String data using the AWS KMS SDK. We assume that
+    /// the SDK has the values required to interact with the AWS KMS APIs (`AWS_ACCESS_KEY_ID` and
+    /// `AWS_SECRET_ACCESS_KEY`) either set in environment variables, or that the SDK is running in
+    /// a machine that is able to assume an IAM role. Moreover the cipher is BASE64 encoded before
+    /// sending the resultant cipher.
```

Comment:
I don't think we need the last sentence, it is an implementation detail. What's important is that we uphold the input/output values in our implementation be directly usable with the other implementations (for example, the AWS CLI).